data/brands/shop/supermarket.json

```diff
@@ -7746,6 +7746,17 @@
         "shop": "supermarket"
       }
     },
+    {
+      "displayName": "Supermarket (Brasil)",
+      "id": "supermarket-20f2a8",
+      "locationSet": {"include": ["br"]},
+      "tags": {
+        "brand": "Supermarket",
+        "brand:wikidata": "Q127375573",
```

Comment:
[This Wikidata item](https://www.wikidata.org/wiki/Q127375573) may need [some additional information](https://wiki.openstreetmap.org/wiki/Key:brand:wikidata#Establishing_notability) on it. Otherwise, a Wikidata administrator may delete it as spam without realizing it’s legitimate.

Reply:
Yeah, unfortunately a Wikipedia article has been created but then deleted last year, with the allegation that the brand "only exists" and therefore is not relevant to be on the encyclopedia.
https://pt.wikipedia.org/wiki/Wikip%C3%A9dia:P%C3%A1ginas_para_eliminar/Rede_Supermarket

Reply:
Added a couple statements to Wikidata item, as well as copied [one photo from Mapillary](https://www.mapillary.com/app/?lat=-22.899917602539&lng=-43.202117919922&z=17.693165920333104&pKey=849641508964035&focus=photo) to [Wikimedia Commons](https://commons.wikimedia.org/wiki/File:Supermarket_na_Rua_Santo_Cristo,_Rio_de_Janeiro.jpg), but for whatever reason I wasn't able to link it on the Wikidata item.